src/NuGet.Core/NuGet.Versioning/NuGet.Versioning.csproj

```
@@ -3,7 +3,7 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary);net45</TargetFrameworks>
```

Avoid `net45` TFM in source-build mode:

```suggestion
    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
    <TargetFrameworks Condition="'$(IsBuildOnlyXPLATProjects)' != 'true'">$(TargetFrameworks);net45</TargetFrameworks>
```

---

fixed in next iteration.